media-video/syncplay/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST syncplay-1.5.0.tar.gz 1588514 BLAKE2B 0f2bddf3500758c3081bdba993b21fe60a0d3af7081c5edd0f684e1d60a2a95edce133a597e79747350e681dd40768226226f0b26e295850a8c218107b1da3e9 SHA512 8a57add8a845c041c02147415fc4b7941a41edfb80fc037c3af4e246d9ca6391be9f3886407470b1b51b8783ad6306e42bb34991e7cbb08eea87e781e3cd2200
```

Comment:
Please state to which version you're bumping in the first line of commit message.